### PR TITLE
[MODINVOICE-504] - Release v5.7.0

### PR DESCRIPTION
**Pull request by Dmitriy-Butramyou**
[MODINVOICE-504](https://issues.folio.org/browse/MODINVOICE-504) - mod-invoice: module release
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate action.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code are 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added, or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,41 @@
-## 5.7.0 - Unreleased
+## 5.8.0 - Unreleased
```
Comment (SerhiiNosko) [inline NEWS.md:1]: the latest version of data-import-converter-storage interface is 1.4, we should update to this version

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,41 @@
...
+
+## 5.7.0 - Released Poppy R2 2023
+The focus of this release was to implement Vertx SFTP client for Batch Voucher Export
```
Comment (SerhiiNosko) [inline NEWS.md:4]: the feature to pay against previous fiscaly year is also important amout of work that should be added here in this description

File: NEWS.md (hunk 1 above)
Comment (Dmitriy-Butramyou) [reply NEWS.md:1]: Version updated

File: NEWS.md (hunk 2 above)
Comment (Dmitriy-Butramyou) [reply NEWS.md]: Added

**Review by SerhiiNosko — APPROVED** [review]

**Review by imerabishvili — APPROVED** [review]

**Review by azizbekxm — APPROVED** [review]

**Review by Abdulkhakimov — APPROVED** [review]

**Pull request merged**